### PR TITLE
Fix switch network

### PR DESCRIPTION
**Pull request by janndriessen**
* Fixes switching network
* Fixes learn more button width on mobile

Based on preview branch. ⚠️

### CODE REVIEW

File: packages/next-app/src/pages/index.tsx [hunk 1]
```diff
@@ -68,6 +68,10 @@ const Home: NextPage = () => {
     typeof accountData !== "undefined" &&
     Object.entries(accountData).length > 0;
 
+  const chainId = networkData.chain?.id ?? -1;
+  const isMainnet = chainId === 1;
+  const isUnsupported = !isMainnet;
```
Comment (embiem) [inline packages/next-app/src/pages/index.tsx:73]: This won't support testing locally or on a testnet, which works for me in the current setup. Can you please elaborate on what's not working for you in the current implementation?

**Comment by embiem** [comment]
This is one of the latest preview deployments using Rinkeby as target network: https://code-claim-site-cuce72z5j-developdao.vercel.app/

If you connect your wallet there from e.g. Mainnet, it should correctly switch Network to Rinkeby. Is this behavior broken for you?

**Comment by embiem** [comment]
In case this is actually not an issue, I'll pick the learn more button width fix into Preview and then merge preview into main.

File: packages/next-app/src/pages/index.tsx (hunk 1 above)
Comment (janndriessen) [reply packages/next-app/src/pages/index.tsx:73]: True. It wouldn't work at all for me but I think I now know why. When I run `yarn dev` I get this:

```
➤ YN0000: Error HH604: Error running JSON-RPC server: ERROR processing /git/code-claim-site/packages/hardhat/deploy/00_deploy_token.ts:
➤ YN0000: Error: ENOENT: no such file or directory, open '/git/code-claim-site/packages/hardhat/artifacts/build-info/02c2c1f1e41187ae532ff288f13c3d95.json'
➤ YN0000: deploying "CODEToken" (tx: 0xcb533865f145c6aeda04f167e45051c6dd0184a44cf0a79ccc256eccd09c6899)...
➤ YN0000: For more info go to https://hardhat.org/HH604 or run Hardhat with --show-stack-traces
```

I see the hardhat config has some env vars. Do they have to be set or is there anything else I have to do to run it locally besides the steps documented unter `Main Scripts`?

@embiem

**Comment by janndriessen** [comment]
@embiem "reverted" the changes but feel free to also just cherry pick that one commit.

File: packages/next-app/src/pages/index.tsx (hunk 1 above)
Comment (embiem) [reply packages/next-app/src/pages/index.tsx]: @janndriessen none of the env vars are necessary for local dev.

The error suggests that it couldn't find the build artifacts... could you try running `yarn build` before yarn dev?

**Pull request merged**